ang/civicase/CaseListTable.html

```diff
@@ -1,5 +1,5 @@
-<table ng-show="!caseIsFocused" class="case-list-table">
-  <thead calculate-scroll-width>
+<table ng-show="!caseIsFocused" class="table civicase__table--caselist">
+  <thead calculate-scroll-width sticky-table-header class="civicase__table--caselist__header">
```

Comment:
Is this class `civicase__table--caselist__header` used anywhere? Remove if not.

Reply:
Used in line _case-list-table.scss